tests/test_api.py

```diff
@@ -58,27 +60,25 @@ def test_create_with_config(self):
         with mock.patch.object(config, '_CONFIG', self.config):
             self.assertEqual(config.get_config(), self.config)
             client = api.Client()
-            cfg_host = self.config['qcs']['hostname']
-            self.assertEqual(cfg_host, client.url.strip('api/v1/'))
+            self.assertEqual(client.url, 'http://example.com/api/v1/')
```

Comment:
Better to have a string as the expected value to make it more clear. Do you agree?

Reply:
That is very readable, yes